TransformationSystem/Client/TaskManagerPlugin.py

```diff
@@ -106,16 +106,18 @@ def _ByJobType( self ):
     destSites = set( res['Value'] )
 
     # 2. get JobTypeMapping "Exclude" value (and add autoAddedSites)
+    gLogger.debug( "Getting JobTypeMapping 'Exclude' value (and add autoAddedSites)" )
     jobType = self.params['JobType']
     if not jobType:
       raise RuntimeError( "No jobType specified" )
     excludedSites = self.opsH.getValue( 'JobTypeMapping/%s/Exclude' % jobType, '' )
+    gLogger.debug( "Excluded sites for %s jobs: %s" % ( jobType, excludedSites ) )
     excludedSites = excludedSites + ',' + ','.join( fromChar( self.opsH.getValue( 'JobTypeMapping/AutoAddedSites', '' ) ) )
 
     # 3. removing sites in Exclude
     if not excludedSites:
       pass
-    elif excludedSites == 'ALL' or 'ALL' in excludedSites:
+    elif excludedSites == 'ALL' or 'ALL,' in excludedSites:
```

Comment:
As I understand you want to have it as a comma-separated string, therefore this is OK to keep it a string but I agree with Chris, it is better to have:
'ALL' in [site.strip().upper() for site in excludedSites.split(',')]
as here you have not considered the case when "ALL" is at the end ("CERN,ALL") ;-)